libs/blocks/caas/utils.js

```diff
@@ -16,17 +16,49 @@ const fetchWithTimeout = async (resource, options = {}) => {
   return response;
 };
 
-export const loadStrings = async (url) => {
-  // TODO: Loc based loading
+export const locales = ['ae_ar', 'ae_en', 'africa', 'ar', 'au', 'be_en', 'be_fr', 'be_nl', 'bg', 'ca', 'ca_fr', 'ch_de',
+  'ch_fr', 'ch_it', 'cis_en', 'cl', 'cn', 'cy_en', 'de', 'es', 'fr', 'gr_en', 'hk_en', 'id_en', 'ie', 'il_en', 'in',
+  'jp', 'kr', 'la', 'langstore', 'lu_de', 'lu_en', 'lu_fr', 'lv', 'mena_ar', 'mena_en', 'mt', 'mx', 'my_en', 'nl',
+  'nz', 'pe', 'ph_en', 'pt', 'sa_ar', 'sa_en', 'se', 'sea', 'sg', 'th_en', 'uk', 'vn_en'];
+
+export function getPageLocale(currentPath){
+  let possibleLocale = currentPath.split('/')[1];
+  for(let locale of locales){
+    if(locale === possibleLocale){
+      return locale;
+    }
+  }
+  // defaults to en_US
+  return '';
+}
+
+export const loadStrings = async (url, pathname=location.pathname) => {
   if (!url) return {};
-  const resp = await fetch(url);
-  if (!resp.ok) return {};
-  const json = await resp.json();
-  const convertToObj = (data) => data.reduce((obj, { key, val }) => {
-    obj[key] = val;
-    return obj;
-  }, {});
-  return convertToObj(json.data);
+  try {
+    let locale = getPageLocale(pathname);
+    if(locale){
+      let _url = new URL(url);
```

Comment:
Please run the linter on this file - you can use `npx eslint libs/blocks/caas/utils.js`